projects/plugins/jetpack/extensions/blocks/premium-content/_inc/subscription-service/class-jetpack-token-subscription-service.php

```diff
@@ -17,13 +18,22 @@
  */
 class Jetpack_Token_Subscription_Service extends Token_Subscription_Service {
 
+	/**
+	 * Is this code executing on WPCOM?
+	 *
+	 * @return bool True if executing on WPCOM.
+	 */
+	private function is_wpcom() {
```

Comment:
Could we just remove this and use `new Host() )->is_wpcom_simple()` everywhere else? It does not add much...